cmd/juju/application/package_test.go

```diff
@@ -12,6 +12,9 @@ import (
 
 // TODO(wallyworld) - convert tests moved across from commands package to not require mongo
```

Comment:
Is this considered done based in this patch?

Reply:
Not yet unfortunately